config/crds/forklift_v1alpha1_networkmap.yaml

```diff
@@ -53,11 +53,15 @@ spec:
                     description: Source network.
                     properties:
                       id:
-                        description: 'The object identifier. For:   - vsphere: The
-                          managed object ID.'
+                        description: 'The object ID. vsphere:   The managed object
+                          ID.'
+                        type: string
+                      name:
+                        description: 'An object Name. vsphere:   A qualified name.'
+                        type: string
+                      type:
+                        description: Kind.
                         type: string
-                    required:
-                    - id
```

Comment:
required `id`  replaced with optional (one of):
- id
- name
- type
validated by the controller.